microprofile-fault-tolerance/pom.xml

```diff
@@ -99,5 +99,130 @@
                 </plugins>
             </build>
         </profile>
+        <!-- Test against Bootable JAR -->
+        <profile>
+            <id>bootablejar.profile</id>
+            <properties>
+                <!-- TBD - remove once https://issues.redhat.com/browse/WFARQ-74 is done and wildfly-arquillian-3.0.0.Beta2 is out -->
+                <jboss.home>${project.build.directory}</jboss.home>
+            </properties>
+            <activation>
+                <property>
+                    <name>ts.bootable</name>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.wildfly.arquillian</groupId>
+                    <artifactId>wildfly-arquillian-container-bootable</artifactId>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
This - as the other following plugins - are already configured at parent level, see https://github.com/fabiobrz/eap-microprofile-test-suite/blob/bootable-jar/pom.xml#L519, so I guess this can be removed.